src/styles/index.scss

```
@@ -1,3 +1,85 @@
+$day--width: 100px;
+$day--height: 100px;
```

No need to duplicate variables with the same values
```suggestion
$day-size: 100px;
```